src/portal/lib/src/tag/tag.component.html

```diff
@@ -58,6 +69,7 @@ <h3 class="modal-title">{{ manifestInfoTitle | translate }}</h3>
             </div>
             </clr-dropdown-menu>
           </clr-dropdown>
+          <button type="button" class="btn btn-sm btn-secondary" [disabled]="!(selectedRow.length==1)" (click)="retag(selectedRow)"><clr-icon shape="copy" size="16"></clr-icon>&nbsp;{{'REPOSITORY.RETAG' | translate}}</button>
           <button type="button" class="btn btn-sm btn-secondary" *ngIf="hasProjectAdminRole" (click)="deleteTags(selectedRow)" [disabled]="!selectedRow.length"><clr-icon shape="times" size="16"></clr-icon>&nbsp;{{'REPOSITORY.DELETE' | translate}}</button>
         </clr-dg-action-bar>
         <clr-dg-column style="width: 120px;" [clrDgField]="'name'">{{'REPOSITORY.TAG' | translate}}</clr-dg-column>
```

Comment:
move this style to scss.

Reply:
this is not introduced by this PR, and there are lots of them here, I'd suggest to fix them together in one dedicated commit.

Reply:
+1

Reply:
ok. this is ok for this pr. Please make sure new introduced style all put in scss.